### ISSUE TITLE
Angular 4 support

### ISSUE THREAD

**Issue by AlexBokhankovich** [yujuiting/ng2-window-view#6]
Is there any chance for angular 4 support?

**Comment by b0bi79** [comment]
https://github.com/b0bi79/ng2-window-view

**Comment by yujuiting** [comment]
No. I had leave Angular development and did not have update plan. Sorry about that. Please feel free to fork.
@AlexBokhankovich
